ContentSecurityPolicy/Violation/Event.php

```diff
@@ -11,7 +11,15 @@
 
 namespace Nelmio\SecurityBundle\ContentSecurityPolicy\Violation;
 
-use Symfony\Component\EventDispatcher\Event as BaseEvent;
+if (\class_exists(\Symfony\Component\EventDispatcher\Event::class)) {
```

Comment:
If we want to avoid deprecation warnings in 4.4, we should change the detection to prefer using the new base class. But this can be a BC break for listeners typehinting `Event`.

Reply:
@stof I would prefer not having BC breaks over a deprecation warning (only in SF 4.4). 

Another fix would be to change this package to only allow supported versions of Symfony Framework and PHP. Maintaining a lot of possible versions will only lead to more similar if-checks.